.github/workflows/cancel_redundant_workflows.yml

```diff
@@ -3,6 +3,7 @@ on:
   workflow_run:
     types:
       - requested
+    # NOTE: Make sure to add to this list as you add more workflows running on 'pull_request'
```

Comment:
lol I feel like no one will read this, but that's OK since we check this in CI :)